theano/gpuarray/opt.py

```diff
@@ -55,14 +59,34 @@
 
 
 gpu_optimizer = EquilibriumDB()
+gpu_optimizer2 = EquilibriumDB()
 gpu_cut_copies = EquilibriumDB()
 
+old_not_transferred = []
+new_not_transferred = []
```

Comment:
Actually, they to not seem to be used, except maybe for debugging, so in that case maybe they should be removed?